### DISCUSSION TITLE
User invitation flow in Ory

### DISCUSSION THREAD

**Discussion by binoiii** [ory/kratos#3688] [Q&A] [answered]
***First option:***
As [recommended](https://www.ory.sh/docs/kratos/manage-identities/invite-users) by the Ory documentation, we can use the API `createRecoveryLinkForIdentity` to get the recovery link.
However, ***we will be the one to implement the sending functionality*** of the link to the user. 

***Second option:***
Make use of the folllowing APIs below.
[createBrowserRecoveryFlow](https://www.ory.sh/docs/reference/api#tag/frontend/operation/createBrowserRecoveryFlow)
[updateRecoveryFlow](https://www.ory.sh/docs/reference/api#tag/frontend/operation/updateRecoveryFlow)
The following APIs allow us to let Ory to natively send the email notification (recovery email)

Is there a way to use the first option and let Ory do the sending of the recovery link? Thanks in advance!

**Comment by vinckr** [comment]
Hello @binoiii 
AFAICT no, but I did not test it now. 

Can you elaborate a bit more on why you want to send the email? 
I assume you would prefer option 1 because there is only one API call involved, is that correct?

**Reply by binoiii** [reply]
Hey @vinckr ,

Thanks for the quick response and asking why we need to send the recovery link via email!

Agree that we prefer option 1 due to it is more straightforward. Hence, the ***recovery link*** must be forwarded via email. Kindly, see the scenario below for our use case.

1. Users with `create-user` permission creates a user.
2. After a user has been created, a ***button will appear to send the invitation link (recovery link) to the email address*** of the created user. 
![image](https://github.com/ory/kratos/assets/54553660/defde312-a93d-4a98-95c2-f27224d1c9b9)
3. There is a ***need to send it via email*** because the ***newly created user doesn't have a permission to view the create user page***.

**Reply by vinckr** [reply, marked as answer]
Hello @binoiii 

Please reach out to your account manager or support@ory.sh to get help for your Ory Network project!
